### ISSUE TITLE
bugbug is to aggressive at moving bugs from "Firefox Build :: General"

### ISSUE THREAD

**Issue by tomprince** [mozilla/bugbug#1483]
That category is a good home for some bugs, and I think bugbug is too aggressive about moving some bugs from there (particularly ones filed by people who work in that area).

For example, :glandium filed https://bugzilla.mozilla.org/show_bug.cgi?id=1628981 in ":: General", and I don't think bugbug should have moved it.

---

Thinking about heuristics to use, I wonder if looking at if the reporter has ever filed bugs in the target component, or moved bugs to the target component, and using that to weight against moving to the target component in the calculation.

**Comment by marco-c** [comment]
The initial idea was to slowly decommission Firefox::General, to have a better home for most bugs.
Maybe the same thinking should only apply to Firefox::General though, and not Firefox Build System::General, as it's already quite a specific component.

> For example, :glandium filed https://bugzilla.mozilla.org/show_bug.cgi?id=1628981 in ":: General", and I don't think bugbug should have moved it.

For this bug in particular, I guess I would have been wrong too :P

**Comment by tomprince** [comment]
> The initial idea was to slowly decommission Firefox::General, to have a better home for most bugs.
> Maybe the same thinking should only apply to Firefox::General though, and not Firefox Build System::General, as it's already quite a specific component.

Yeah, the idea of breaking up Firefox::General makes sense, and I suspect bugs get filed in Firefox Build System::General that should be in more specific components, but some things are just build system bugs, and there is not a more specific place to file them.

> 
> > For example, :glandium filed https://bugzilla.mozilla.org/show_bug.cgi?id=1628981 in ":: General", and I don't think bugbug should have moved it.
> 
> For this bug in particular, I guess I would have been wrong too :P

I definitely could see an argument that it belongs in Task Configuration, but the reason I (and I guess Mike) didn't put it there is because it requires build system changes in addition to task changes.

**Comment by sylvestre** [comment]
> For this bug in particular, I guess I would have been wrong too :P

Same, when you changed it back to General, I was confused too.
In general(!), General are just like untriagged...
